### PR TITLE
Added command for Studio unit test status.

### PR DESCRIPTION
**Pull request by dggriffin**
> studbot unit_test status
> Command to get Studio unit test status.

### CODE REVIEW

File: scripts/unit_test.coffee [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = (robot) ->
+
+	robot.respond /unit_test status/i, (msg) ->
+		robot.http("http://vbujenkins.vistaprint.net/job/Studio.JavaScript.UnitTests/lastBuild/api/json")
+  			.get() (err, res, body) ->
+  				data = JSON.parse(body)
+  				message = "\nLATEST UNIT TEST STATUS:\n" + data.fullDisplayName + ': ' + data.result + ''
+  				if(data.result == 'FAILURE')
```
Comment (TerribleDev) [inline scripts/unit_test.coffee:8]: I'm not sure why you think this is going to work since heroku is hosted outside the company

**Comment by TerribleDev** [comment]
Also exposing internal DNS names to the public Internet is a big no no you should probably injector this as an environment variable or host this repo internally

**Comment by dggriffin** [comment]
@tparnell8 We're looking at moving studbot off of heroku onto our intranet, but you're absolutely right about the internal dns.

**Comment by TerribleDev** [comment]
if you want jenkins to talk to hipchat, we have already installed a plugin into jenkins to do so, so you don't even need a bot to send failure notifications to hipchat

**Comment by dggriffin** [comment]
Ah fo'real? Lame. I deleted the branch and rolled back history.

**Pull request merged**